automatic/kubernetes-cli/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,28 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageName = 'kubernetes-cli'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$installLocation = "$(Get-ToolsLocation)\$packageName"
+
+$packageArgs1 = @{
+  PackageName    = $packageName
+  FileFullPath   = Get-Item $toolsPath\*-386.tar.gz
+  FileFullPath64 = Get-Item $toolsPath\*-amd64.tar.gz
+  Destination    = $installLocation
+}
+Get-ChocolateyUnzip @packageArgs1
+
+$packageArgs2 = @{
+  PackageName    = $packageName
+  FileFullPath   = Get-Item $installLocation\*-386.tar
+  FileFullPath64 = Get-Item $installLocation\*-amd64.tar
+  Destination    = $installLocation
+}
+Get-ChocolateyUnzip @packageArgs2
```

Comment:
wasn't this fixed in a later version of chocolatey?
So you didn't have to extract it twice, could be useful to make a check if it's already extracted or not.

Reply:
Still not fixed in 0.10.8, I will future proof it then

Reply:
I see, I thought it was.